Gruntfile.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(grunt) {
+  grunt.initConfig({
+    jshint: {
+      all: ['Gruntfile.js', 'index.js']
```

Comment:
@Ian-Stewart-Binks sure I can change it to use a regexp to look at all .js files in the root directory (i.e. `all: ['Gruntfile.js', 'index.js']` to `all:['*.js']` 

@emmettu Unfortunately we can't since JSHint doesn't support linting JSX :(